.github/workflows/pr-check.yaml

```diff
@@ -25,66 +25,38 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        include:
-          - os: "windows-2022"
-          - os: "macos-14"
-          - os: "ubuntu-22.04"
-    timeout-minutes: 20
-    env:
-      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        os: [windows-2022, ubuntu-22.04, macos-14]
+    timeout-minutes: 15
```

Comment:
it seems it's not related to yarn/pnpm --> another PR

Reply:
Restored